src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -739,24 +760,28 @@ def fuse_module_conv_bn_relus(
 
 def prepare_embeddings_qat(
     module: Module,
-    qproperties: QConfigProperties,
+    qproperties: Optional[QConfigProperties] = None,
     qconfig: Optional["torch.quantization.QConfig"] = None,
 ):
     """
     adds a fake quantize call to the weights of any Embedding modules in the given
-    module
+    module. The used qconfig will have a heirarchy of
+
+    submodule.qconfig -> qconfig -> qproperties
 
     :param module: module to run QAT for the embeddings of
-    :param qconfig: qconfig to generate the fake quantize ops from. Default uses INT8
-        asymmetric range
-    :param qproperties: properties used to define QConfig.
+    :param qconfig: qconfig to generate the fake quantize ops from if qconfig
+        not set in moduleDefault uses INT8 asymmetric range
+    :param qproperties: properties used to define QConfig if qconfig not present
     """
-    if qconfig is None:
+    if qconfig is None and qproperties is not None:
         qproperties.symmetric_weights = False
         qconfig = get_qat_qconfig(qproperties)
     for submodule in module.modules():
-        if type(submodule) is Embedding:
-            _prepare_qat_embedding(submodule, qconfig)
+        submodule_qconfig = getattr(submodule, "qconfig", None)
+        submodule_qconfig = submodule_qconfig or qconfig
+        if type(submodule) is Embedding and submodule_qconfig is not None:
```

Comment:
`submodule_qconfig is not None` will always be true now right?

```suggestion
        if isinstance(submodule, Embedding):
```

Reply:
`isinstance` allows for subclasses to pass as well, we want to do a hard check that it is actually an `Embedding` since we overwrite the forward pass